### PR TITLE
simplify env vars

### PR DESCRIPTION
**Pull request by ghepting**

### CODE REVIEW

File: apps/google-analytics-4/lambda/.env [hunk 1]
```diff
@@ -4,7 +4,7 @@ SIGNING_SECRET: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 SENTRY_DSN: https://b8f524eae7c446fb8071476431426640@o2239.ingest.sentry.io/4504725335834624
 STAGE: test
 RELEASE_VERSION: test
-AWS_REGION: us-east-1
+DYNAMODB_REGION: us-east-1
```
Comment (ghepting) [inline apps/google-analytics-4/lambda/.env:7]: apparently AWS_REGION is a reserved keyword, so using a custom env var

**Review by jsdalton — APPROVED** [review]
💯

**Review by mgoudy91 — APPROVED** [review]
🆒

File: apps/google-analytics-4/lambda/.env (hunk 1 above)
Comment (mgoudy91) [reply apps/google-analytics-4/lambda/.env:7]: ahhh

**Pull request merged**